users/serializer.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import  serializers
+
+from .models import User
+
+
+class RegisterSerializer(serializers.ModelSerializer):
```

Comment:
shouldn't Serializer name reflect name of model it is serializing?

Reply:
I have updated the name to UserSerializer.